src/components/Icons/SnowflakeCircle.tsx

```diff
@@ -3,28 +3,35 @@ import { useTranslation } from 'react-i18next';
 import styled from 'styled-components';
 import { withStyles, Tooltip } from '@material-ui/core';
 
+import { CurrencyKey } from 'constants/currency';
 import { ReactComponent as SnowflakeIcon } from 'assets/images/snowflake.svg';
+import { darkTheme } from 'styles/theme';
 
 type SnowflakeCircleProps = {
-	radius: number;
-	innerRadius: number;
-	showTooltip: boolean;
-	name?: string;
+	radius?: number;
+	innerRadius?: number;
+	showTooltip?: boolean;
+	currencyKey?: CurrencyKey;
 };
 
-const SnowflakeCircle: FC<SnowflakeCircleProps> = ({ radius, innerRadius, showTooltip, name }) => {
+const SnowflakeCircle: FC<SnowflakeCircleProps> = ({
+	radius = 20,
+	innerRadius = 16,
+	showTooltip = true,
+	currencyKey,
+}) => {
 	const { t } = useTranslation();
-	const FrozenIcon = (
-		<SnowflakeCircleWrapper radius={radius}>
-			<StyledSnowflakeCircle innerRadius={innerRadius} />
-		</SnowflakeCircleWrapper>
-	);
-	return showTooltip ? (
-		<StyledTooltip title={<>{t('common.frozen.message', { name })}</>} placement="top" arrow={true}>
-			{FrozenIcon}
+
+	return (
+		<StyledTooltip
+			title={showTooltip ? <>{t('common.frozen.message', { currencyKey })}</> : ''}
```

Comment:
if you pass a title with zero length, the tooltip will not render anything (this is a way to "hide" it)